### ISSUE TITLE
When building and running tests I get `Process exited with code 255`

### ISSUE THREAD

**Issue by Sibz** [JetBrains/teamcity-unity-plugin#44]
Unity 2020.1.0b4 (Also tried 2019.3.08f)
Latest Teamcity
Running Windows Version 10.0.18363

The build without tests succeeds, however when I run with tests I get an error. I've tried to figure out what it is but I am at a loss. Oh and the tests are all passing.

The repository I am testing is here:
https://github.com/Sibz/Com.Sibz.NetCode

I basically have a default TeamCity install and only set this unity build settings:
Build Target: standalone
Player: Win64
Output Path: Builds

Full Log: Added in comment below

Tail of log (I snipped the times except for 1 line so it can be matched to full log):
```
  AcceleratorClientConnectionCallback - disconnected - :0
  Cleanup mono
  Checking for leaked weakptr:
    Found no leaked weakptrs.
[22:48:17]  [Package Manager] Server::Kill -- Server was shutdown
  Process exited with code 255
  Process exited with code 255 (Step: Unity)
  Importing data from 'C:\TeamCity\buildAgent\temp\buildTmp\unityTestResults-1851156133729896712.xml' (31.05 KB) with 'nunit' processor
  NUnit report watcher
  Waiting for 1 service processes to complete
  NUnit report watcher
    1 report found for paths:
    C:\TeamCity\buildAgent\temp\buildTmp\unityTestResults-1851156133729896712.xml
    Successfully parsed
      1 report
      C:\TeamCity\buildAgent\temp\buildTmp\unityTestResults-1851156133729896712.xml
  Step Unity failed
Publishing internal artifacts
Build finished
```

**Comment by Sibz** [comment]
[Netcode_Build_20.log.txt](https://github.com/JetBrains/teamcity-unity-plugin/files/4438549/Netcode_Build_20.log.txt)
Full Log

**Comment by Sibz** [comment]
I have solved this, but will post my solution here for prosperity:
I am using the Unity NetCode package. But to test in edit mode I needed to run the boot strapper manually, also I figured I needed it to be an editor world so I had the following code in a OneTimeSetup method:
```c#
DefaultWorldInitialization.Initialize("DefaultTestWorld", true);
new ClientServerBootstrap().Initialize("DefaultNetWorld");
```
Not sure if it's because it was an editor world, or because I am initializing the default world twice. But I had found out already the world doesn't update in editor mode unless the editor is doing something, so I was manually updating the world systems for test anyway. I removed the first line and the Exit code 255 went away.
